schema.graphql

```diff
@@ -12,6 +12,16 @@ input EventFilterOptionsInput {
   from: Int
 }
 
+input ActionFilterOptionsInput {
+  address: String!
+  tokenId: String
+  status: BlockStatusFilter
+  to: Int
+  from: Int
+  fromActionHash: String
```

Comment:
could we rename this to `fromActionState` / `endActionState` for consistency with the rest of the protocol? I'm going to argue for the same on the SnarkyJS side